packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -12,6 +12,7 @@
 
   --calcite-min-header-height: calc(var(--calcite-icon-size) * 3);
   --calcite-panel-footer-padding: theme("spacing.2");
+  --calcite-internal-panel-header-border-block-end: 1px solid var(--calcite-ui-border-3);
```

Comment:
Can the `-internal` be at the end? I think that's the pattern elsewhere - maybe we can add that to conventions? https://github.com/Esri/calcite-design-system/blob/bb07831eb518d1d617b765e1f14279480c6051fc/packages/calcite-components/src/components/chip/chip.scss#L3

Reply:
☝️ this is actually inconsistent with some other internal CSS props. 😅

I think `internal` should be more prominent in the name. This might be easier to spot if it's part of the prefix.

Reply:
> maybe we can add that to conventions?

Excellent idea. I don't think we have this documented at the moment.

Reply:
> 

LOL - yeah it's about 50/50. I feel like it's easier to spot at the end with nothing following it but... as long as its consistent either way.

Reply:
I feel like we should follow events as they are prefixed with `calciteInternal`.

Reply:
🚢  it